### PR TITLE
HIPP-1541: Display the no. of APIs deployed in prod

### PR DESCRIPTION
**Pull request by PaulCDurham**
https://jira.tools.tax.service.gov.uk/browse/HIPP-1541

### CODE REVIEW

File: app/uk/gov/hmrc/apihubapplications/controllers/StatsController.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  cc: ControllerComponents,
+  identify: IdentifierAction,
+  statsService: StatsService
+)(implicit ec: ExecutionContext) extends BackendController(cc) {
+
+  def apisInProduction(): Action[AnyContent] = identify.async {
+    implicit request =>
+      statsService.apisInProduction().map {
+        case Right(statistic) => Ok(Json.toJson(statistic))
+        case Left(e) => throw e
```
Comment (victorarbuesmallada) [inline app/uk/gov/hmrc/apihubapplications/controllers/StatsController.scala:39]: I guess this is the same as returning an `InternalServerError`?

File: test/uk/gov/hmrc/apihubapplications/controllers/StatsControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+      running(fixture.application) {
+        val request = FakeRequest(routes.StatsController.apisInProduction())
+        val result = route(fixture.application, request).value
+
+        status(result) mustBe OK
+        contentAsJson(result) mustBe Json.toJson(statistic)
+      }
+    }
+  }
```
Comment (victorarbuesmallada) [inline test/uk/gov/hmrc/apihubapplications/controllers/StatsControllerSpec.scala:43]: is it worth having a case for when it the `StatsService` fails?

**Review by victorarbuesmallada — APPROVED** [review]

File: app/uk/gov/hmrc/apihubapplications/controllers/StatsController.scala (hunk 1 above)
Comment (PaulCDurham) [reply app/uk/gov/hmrc/apihubapplications/controllers/StatsController.scala:39]: I don't like returning Internal Server Error as this bypasses Play's error handler and we don't get good stuff like logging. I think we should raise a ticket at some point to get rid of all that and stick with `throw e`.

Comment (PaulCDurham) [reply app/uk/gov/hmrc/apihubapplications/controllers/StatsController.scala:39]: But yes, it has the same result. The frontend will get a 500 Internal Server Error response.

**Pull request merged**